packages/client/src/views/RegisterForm/RegisterForm.tsx

```diff
@@ -686,13 +686,17 @@ class RegisterFormView extends React.Component<FullProps, State> {
         informant = 'father'
       }
 
+      // informant needs to be reset as the relationship changed
+      // see https://github.com/opencrvs/opencrvs-core/issues/5866
       if (informant) {
+        const { _fhirIDPatient, ...informantWithoutFhirID } =
+          declaration.data[informant]
```

Comment:
```suggestion
          declaration.data.informant
```
The informant & father/mother details need to be handled separately